gpt.py

```diff
@@ -0,0 +1,63 @@
+import os
+import openai
+import asyncio
+from datetime import datetime, timedelta
+from dotenv import load_dotenv
+
+# Load environment variables from .env file
+load_dotenv()
+
+# Retrieve OpenAI API key from environment variable
+KEY = os.environ.get("KEY")

```

Comment:
You should give you openai api key a more indicative name.
what will you do if you need to save other keys as a programmer?